pkg/kubelet/dockertools/manager.go

```diff
@@ -605,6 +605,7 @@ func (dm *DockerManager) runContainer(
 			MemorySwap: -1,
 			CPUShares:  cpuShares,
 		},
+		ShmSize:     1024 * 1024 * 64,
```

Comment:
@ncdc Alright. Docker doesn't support ShmSize of 0 after https://github.com/anusha-ragunathan/docker/commit/ef1d410b0270fa7309d76aded34113396def7fb2, but change to support that after https://github.com/docker/docker/commit/5190794f1d85d5406611eb69c270df62ac1cdc7f.

There is no docker version released between them, so I think as you said, we don't need to care about it for now. :)